docker-compose.yml

```diff
@@ -34,14 +34,15 @@ services:
     logging: *log-config
 
   bahmni-lab:
+    profiles: ["lab-lite","lite"]
     image: 'bahmni/bahmni-lab:${BAHMNI_LAB_IMAGE_TAG:?}'
     ports:
       - '8090:80'
       - '8443:443'
     logging: *log-config
 
   openelis:
-    profiles: ['default', 'openelis']
+    profiles: ["openelis","standard"]
```

Comment:
Instead of `standard` can we call the profile `bahmni-standard` (so bahmni gets free advertisement!). 
Ditto instead of `lite` can we call the profile `bahmni-lite`.

So the command would be `docker-compose --profile bahmni-lite up`.